src/main.py

```diff
@@ -48,42 +57,118 @@ async def root():
 # Could look at 'get_forecast_metadata' to see when this forecast is made
 
 # get_sites: Clients get the site id that are available to them
-@app.get("/sites/site_list")
-async def get_sites(pv_sites: PV_Sites):
-    return {"pv_sites": PV_Sites}
+@app.get("/sites/site_list", response_model=PV_Sites)
+async def get_sites(site_uuid: str):
+    # simple 2. (fake just return a list of one site using 'fake_site_uuid'
+    pv_site = PV_Site_Metadata(
+        uuid=site_uuid,
+        site_name="fake site name", 
+        latitude= 50,
+        longitude= 0,
+        capacity_kw= 1 
+    )
+    pv_site_list = PV_Sites(
+        site_list= [pv_site],
+    )
+
+    return pv_site_list
 
 
 # # post_pv_actual: sends data to us, and we save to database
-@app.post("/sites/pv_actual/{site_id}")
-async def post_pv_actual(one_pv_actual: One_PV_Actual):
-    return {"one_pv_actual": One_PV_Actual}
+@app.post("/sites/pv_actual/{site_id}", response_model=One_PV_Actual)
+async def post_pv_actual(site_uuid:str, datetime_utc:str, actual_generation_kw:float):
```

Comment:
good try, the user should pass the actual the ONE_PV_Actual object. In the real situation, we would save this to a database. 

Perhaps we dont even need to return anything here either .... (thats my mistake)

Reply:
I'll update this for you, and you can do the one below

Reply:
The only reason we might have something returned is if we'll need the information on the front end  for some reason, which is what I was thinking (without knowing how or why that might be useful). will users be inputing their data through a UI? our current UI doesn't seem to have that as an option. we could also return a message that the data was input correctly.

Reply:
actually 204 might be better